polyphemus/lib/commands.rb

```diff
@@ -504,6 +504,225 @@ def execute(env, project_name, file_path, execute: false)
     end
   end
 
+  class PullGNESharedData < Etna::Command
+    include WithEtnaClients
+    include WithLogger
+
+    string_flags << '--out-dir'
+    boolean_flags << '--stub-files'
+
+    MATERIALIZE_ATTRIBUTES = {
+        'patient' => [
+            'created_at',
```

Comment:
super-nit: does GNE need the `created_at` and `updated_at` fields for all these models? Those aren't the source timestamps (i.e. REDCap), but rather Magma timestamps.

Reply:
Not 100% sure, but this list came from Gabi herself.   We can't currently send the REDCap values in any case since these values are updated automatically as part of magma logic.

At some point I imagine us formalizing the idea that magma is not a source of truth, but a composition of sources of truth, in which case values like these could be pass through to their source and not owned by magma itself.

For now, any value which atleast reflects for the purpose of GNE wether data is 'new' or 'updated' seems useful.

Reply:
> For now, any value which atleast reflects for the purpose of GNE wether data is 'new' or 'updated' seems useful.
:+1: